tests/testthat/test-query.R

```diff
@@ -190,3 +190,27 @@ test_that("get_metadata() is cached", {
     
     identical(table, table_2) |> expect_true()
 })
+
+test_that("database_url() expect character ", {
+  DATABASE_URL() |>
+    expect_s3_class("character")
+})
+
+test_that("get_metadata() returns fibrosis data when specify", {
+  fibro_url = DATABASE_URL("fibrosis.0.2.3.parquet")
```

Comment:
Would be best to use `<-` for assignment for consistency with the rest of the codebase.
```suggestion
  fibro_url <- DATABASE_URL("fibrosis.0.2.3.parquet")
```